src/main/plugins/org.dita.html5/xsl/functions.xsl

```diff
@@ -137,6 +146,66 @@ See the accompanying LICENSE file for applicable license.
        table:get-current-tgroup($el)/@rowsep)[1]
     "/>
   </xsl:function>
+  
+  <xsl:function name="table:find-entry-end-column" as="xs:integer">
+    <xsl:param name="ctx" as="element()"/>
+    <xsl:choose>
+      <xsl:when test="$ctx/@nameend">
+        <xsl:value-of select="xs:integer(table:get-ending-colspec($ctx)/@colnum)"/>
```

Comment:
IMO it's better to use `xsl:sequence` when returning an integer, because `xsl:value-of` will return a string value and the function return type will then convert it back to integer